pages/index.tsx

```diff
@@ -10,4 +11,12 @@ export default function HomePage() {
             <SupportForm />
         </>
     )
+}
+
+export async function getStaticProps({ locale }:any) {
```

Comment:
```suggestion
export async function getStaticProps({ locale }: { locale: string } ) {
```

locale is string
just check what `serverSideTranslations()` takes as first argument by hovering on it

<img width="581" alt="image" src="https://user-images.githubusercontent.com/61631665/201541356-a270f4de-f27f-4ef1-a5e6-c13d18c8e8d2.png">